BiasAndFairnessServers/src/controllers/bias_and_fairness.py

```
@@ -0,0 +1,86 @@
+                name=data_filename,
+                target_column=target_column,
+                sensitive_column=sensitive_column,
+                model_id=upload_model_record.id,
+                db=db
+            )
+
+            if not upload_data_record:
+                raise Exception("failed to upload data file")
+
+            result = analyze_fairness(
+                model_content=model_content, 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)